cli/src/config-schema.json

```diff
@@ -183,6 +183,20 @@
                         "bright white"
                     ]
                 },
+                "hexColor": {
+                    "type": "string",
+                    "pattern": "^#[0-9a-fA-F]{6}$"
+                },
+                "colors": {
+                  "oneOf": [
+                    {
+                        "$ref": "#/properties/colors/definitions/colorNames"
+                    },
+                    {
+                        "$ref": "#/properties/colors/definitions/hexColor"
+                    }
+                  ]
+                },
```

Comment:
I don't really know a great way to check this schema, so apologies if I've got it totally wrong.